Revit_Core_Engine/Query/LookupParameter.cs

```diff
@@ -51,9 +55,14 @@ public static Parameter LookupParameter(this Element element, IEnumerable<string
             {
                 foreach (Parameter p in element.Parameters)
                 {
+#if REVIT2020 || REVIT2021 || REVIT2022 || REVIT2023 || REVIT2024
+                    BuiltInParameterGroup groupType = p.Definition.ParameterGroup;
+#else
+                    ForgeTypeId groupType = p.Definition.GetGroupTypeId();
+#endif
```

Comment:
Couldn't this become something around `var groupType = p.Definition.GroupTypeId()`?

Reply:
Yes that's true, personally I treated such cases as 'different' - most likely wrongly, should be avoiding exceptions. Then maybe an alternative, `dynamic` instead of `var`?